e2e/cypress/integration/team_settings/teams_spec.js

```diff
@@ -211,4 +220,168 @@ describe('Teams Suite', () => {
+
+        // # Change team description in the input
+        cy.get('#teamOpenInvite').click();
+
+        // Save new team description
+        cy.findByText(/save/i).click();
+
+        // # Close the team settings
+        cy.get('body').type('{esc}', {force: true});
+
+        // # Login as new user
+        cy.apiLogin(newUser);
+        cy.visit('/');
+
+        // * Verify if the user is redirected to the Select Team page
+        cy.url().should('include', '/select_team');
+        cy.get('#teamsYouCanJoinContent').should('be.visible');
+
+        // # Select test team name
+        cy.findByText(testTeam.display_name, {timeout: TIMEOUTS.HALF_MIN}).should('be.visible').click();
+        ignoreUncaughtException();
+
+        // # Verify Town square is visible
+        cy.url().should('include', `/${testTeam.name}/channels/town-square`);
+        cy.findByText('Beginning of Town Square').should('be.visible');
+    });
+
+    it('MM-T2322 Do not allow anyone to join this team', () => {
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // # Open the hamburger menu
+        cy.findByLabelText('main menu').should('be.visible').click();
+
+        // # Click on team settings menu item
+        cy.findByText('Team Settings').should('be.visible').click();
+
+        // # Click on the team description menu item
+        cy.findByText('Allow any user with an account on this server to join this team').should('be.visible').click();
+
+        // # Change team description in the input
+        cy.get('#teamOpenInviteNo').click();
+
+        // Save new team description
+        cy.findByText(/save/i).click();
+
+        // # Close the team settings
+        cy.get('body').type('{esc}', {force: true});
+
+        // # Login as new user
+        cy.apiLogin(testUser);
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+        cy.wait(TIMEOUTS.ONE_SEC);
+
+        // # Open the hamburger menu
+        cy.findByLabelText('main menu').should('be.visible').click();
+
+        // # Click on Join another team menu item
+        cy.findByText('Join Another Team').should('be.visible').click();
+
+        // # Verify the original test team isn't on the lit
+        cy.get('.signup-team-dir').children().should('not.contain', `#${testTeam.name.charAt(0).toUpperCase() + testTeam.name.slice(1)}`);
+    });
+
+    it('MM-T2328 Member can view and search for members', () => {
+        cy.apiLogout();
+        cy.wait(TIMEOUTS.ONE_SEC);
+
+        // * Login as test user and join town square
+        cy.apiLogin(testUser);
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // # Open the hamburger menu
+        cy.findByLabelText('main menu').should('be.visible').click();
+
+        // # Click on Join another team menu item
+        cy.findByText('View Members').should('be.visible').click();
+        cy.wait(TIMEOUTS.HALF_SEC);
+        cy.get('#searchUsersInput').type('sysadmin');
```

Comment:
Non-blocking but in the future, please use the returned value from `getAdminAccount()` so we can run this test against a remote server with a different admin account.
```suggestion
        cy.get('#searchUsersInput').type('sysadmin');
```